docs/.templates/integration/schema.json

```diff
@@ -0,0 +1,102 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "title": "Netdata Integrations file",
+    "properties": {
+        "categories": {
+            "type": "object",
+            "description": "A list defining all the available categories for the integrations.",
+            "properties": {
+                "list": {
+                    "type": "array",
+                    "items": {
+                        "type": "object",
+                        "description": "The concept of IDs between categories is used, so we can handle nested categories. That way they will be able to mention their parent, and we should be able to make the structure of categories. Every category's ID will be used by the various integrations too to mention in which category they lie. Along with this, a priority field is needed to define the ordering on a menu-level.",
+                        "properties": {
+                            "name": {
+                                "type": "string",
+                                "description": "Name of the category. This is the actual name of the category or subcategory, (e.g. Web Servers, Operating Systems, Databases)."
+                            },
+                            "priority": {
+                                "type": "integer",
+                                "description": "Priority of the category. A number expressing where the category should be in the menu. The smaller the number the higher the priority, so a category with priority 1 would be above a category with a priority of 50 in terms of menu ordering."
```

Comment:
Hmm, so we prefer some sort of custom menu order, rather than alphabetical? This is left-hand menu under Data Collection, right?

Reply:
Somewhat like this, we don't know where the ordering will live, the app will present them in a different way I think, an integration might belong to X amount of categories

Reply:
> an integration might belong to X amount of categories

Ok, how is that connected with category priority?